win32ss/gdi/ntgdi/freetype.c

```diff
@@ -773,6 +771,71 @@ WeightFromStyle(const char *style_name)
     return FW_NORMAL;
 }
 
+BOOL FASTCALL
+DeleteFontSubstFromList(
+    PLIST_ENTRY     pHead, 
+    PUNICODE_STRING pTargetFaceNameW)
+{
+    PLIST_ENTRY         pListEntry;
+    PFONTSUBST_ENTRY    pSubstEntry;
+    PUNICODE_STRING     pFrom;
+    BOOL                ret = FALSE;
+
+    IntLockFreeType;
+    for (pListEntry = pHead->Flink;
+         pListEntry != pHead;
+         pListEntry = pListEntry->Flink)
+    {
+        pSubstEntry =
+            (PFONTSUBST_ENTRY)CONTAINING_RECORD(pListEntry, FONT_ENTRY, ListEntry);
```

Comment:
This is not correct (there should not be a cast required for this).
Should be:
`pSubstEntry = CONTAINING_RECORD(pListEntry, FONTSUBST_ENTRY, ListEntry);`

Reply:
OK.